spec/src/main/asciidoc/servlet-spec-body.adoc

```diff
@@ -277,8 +271,7 @@ the servlet under the compliance with RFC 7234.
 
 A servlet container may place security
 restrictions on the environment in which a servlet executes. In a Java
-Platform, Standard Edition (J2SE, v.1.3 or above) or Java Platform,
-Enterprise Edition (Java EE, v.1.3 or above) environment, these
+Platform, Standard Edition (J2SE, v.1.3 or above), these
```

Comment:
Should we change this to be 1.8 or above?

Reply:
I'd remove references to Java versions here and just say the security restrictions used by the Java platform may be used.